pages/windows/reg-query.md

```diff
@@ -1,24 +1,36 @@
 # reg query
 
-> Display the values of keys and sub keys in the registry.
+> Display the values of keys and subkeys in the registry.
 > More information: <https://learn.microsoft.com/windows-server/administration/windows-commands/reg-query>.
 
 - Display all values of a key:
 
 `reg query {{key_name}}`
 
-- Display a specific value of a key:
+- Display a specific [v]alue of a key:
 
 `reg query {{key_name}} /v {{value}}`
 
-- Display all values of a key and its sub keys:
+- Display all values of a key and its [s]ubkeys:
 
 `reg query {{key_name}} /s`
 
-- Search for keys and values matching a specific pattern:
+- Search [f]or keys and values matching a specific pattern:
 
 `reg query {{key_name}} /f "{{query_pattern}}"`
 
-- Display a value of a key matching a specified data type:
+- Display a value of a key matching a specified data [t]ype:
 
-`reg query {{key_name}} /t {{type}}`
+`reg query {{key_name}} /t REG_{{SZ|MULTI_SZ|EXPAND_SZ|DWORD|BINARY|NONE}}`
+
+- Only search in [k]ey names or in [d]ata:
+
+`reg query {{key_name}} /{{k|d}}`
```

Comment:
https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/reg-query

> /k
> Specifies to search in key names only. ***Must be used with /f***.

There is no `/f` in this example.

> /d
> Specifies to search in data only.

Since `/k` requires `/f`, and `/d` does not, I would suggest splitting this into two examples.
```suggestion
- Only search in [k]ey names or in [d]ata:

`reg query {{key_name}} /{{k|d}}`
```